docs/src/pages/landing-page/Index.vue

```diff
@@ -134,13 +134,55 @@
           <div class="lp-heading lp-heading-large">
             What our community thinks of quasar
           </div>
-          <div class="text-size-20 text-lp-light text-weight-medium">
+          <div class="lp-heading lp-heading-small">
             "You've never heard of Quasar? It's the framework that made a mobile
-            app, desktop app and browser extension altogether in less than
-            12minutes."
+            app, desktop app and browser extension altogether in less than 12 minutes."
           </div>
         </div>
       </div>
+      <q-carousel
+        v-model="slide"
+        transition-prev="scale"
+        transition-next="scale"
+        swipeable
+        animated
+        navigation
+        :autoplay="true"
+        padding
+        arrows
+        height="400px"
+        class="bg-transparent rounded-borders"
+      >
+        <template v-slot:navigation-icon="{ active, btnProps, onClick }">
+          <q-btn
```

Comment:
These navigation dots should actually be the same size AFAIK
Should also be easier to write the template

```vue
          <q-btn
            size="xs"
            :icon="btnProps.icon"
            :color="active ? 'lp-primary' : 'grey-8"
            flat
            round
            dense
            @click="onClick"
          />
```